all_seaing_interfaces/msg/BuoyPair.msg

```diff
@@ -0,0 +1,3 @@
+geometry_msgs/Pose left
+geometry_msgs/Pose right
+Waypoint waypoint
```

Comment:
Why are the buoys `Pose`s as opposed to `Obstacle`s? I believe that is how we're representing the buoys everywhere else in the code, and you wouldn't need to create the quaternions in that case (I think). 

Also is waypoint just the point btwn the pair